### DISCUSSION TITLE
[Weekly Review Report]: April 8 - 14

### DISCUSSION THREAD

**Discussion by IsabelWassink** [wikipathways/wikipathways-help#113] [curation-reports]
### Recent changes

 * Some small changes made by the curation team

### Under construction

 * Multiple pathways under construction that are still less than 30 days old
 * [WP5458](https://classic.wikipathways.org/index.php/Pathway:WP5458): tagged as "test pathway" but doesn't look like part of the tutorial pathways

### New pathways

 * [WP5462](https://classic.wikipathways.org/index.php/Pathway:WP5462): under construction

### Deleted Pathways

 * [WP5456](https://classic.wikipathways.org/index.php/Pathway:WP5455) and [WP5456](https://classic.wikipathways.org/index.php/Pathway:WP5456) are tagged for deletion and have no recent edits

### Update Approved pathway tag

- [X] All Approved pathways tags were updated!

**Comment by khanspers** [comment]
Thanks @IsabelWassink! I have deleted WP5455 and WP5456.
